install/kots/manifests/gitpod-installer-job.yaml

```diff
@@ -182,16 +182,39 @@ spec:
               config=$(cat "${CONFIG_FILE}")
               echo "Gitpod: ${CONFIG_FILE}=${config}"
 
-              /app/installer render -c "${CONFIG_FILE}" --namespace {{repl Namespace }} > /tmp/gitpod.yaml
+              echo "Gitpod: Create a Helm template directory"
+              rm -Rf "${GITPOD_OBJECTS}"
+              mkdir -p "${GITPOD_OBJECTS}/templates"
+              cat <<EOF >> "${GITPOD_OBJECTS}/Chart.yaml"
+              apiVersion: v2
+              name: gitpod-kots
+              description: Always ready-to-code
+              Version: "1.0.0"
```

Comment:
question: The version here does not matter because this is not user facing from Replicated? and they never directly deal with the chart? Makes sense that way.

Reply:
Correct, the `appVersion` is the important bit which, crucially, doesn't have to be in semver. This is the chart version which does have to be semver. If we ever significantly change the chart (I doubt it, as it's just the `Chart.yaml`) then this number may be incremented